ws/templates/trips/__create_or_edit.html

```diff
@@ -120,6 +120,10 @@ <h2>Signup</h2>
 
   </div>
 
+  <div class="alert alert-warning">
```

Comment:
I'm not sure `warning` is necessarily the level we'd want -- that's typically meant to convey to the user "hey, something's kind of wrong here you should proceed with caution."

This is probably better suited as `alert-info` if we want an alert!